### PR TITLE
add ops for train unet in kunlun

### PR DESCRIPTION
**Pull request by taixiurong**
### PR types
New features

### PR changes
OPs

### Describe
1. fix elementwise ops bug 2. fix softmax_with_cross_entropy_op bug in xpu place 3. add bilinear interp op in xpu
*test=kunlun

### CODE REVIEW

File: paddle/fluid/operators/softmax_with_cross_entropy_op_xpu.cc [hunk 1]
```diff
@@ -70,7 +70,8 @@ class SoftmaxWithCrossEntropyXPUKernel : public framework::OpKernel<T> {
                                      r));
     } else {
       Tensor labels_int32;
-      labels_int32.mutable_data<int32_t>(context.GetPlace(), labels->numel());
+      labels_int32.mutable_data<int32_t>(context.GetPlace(),
```
Comment (QingshuChen) [inline paddle/fluid/operators/softmax_with_cross_entropy_op_xpu.cc:73]: softmax with cross entropy op xpu的单测一起更新下

Comment (taixiurong) [reply paddle/fluid/operators/softmax_with_cross_entropy_op_xpu.cc:73]: 这个 soft_cross_entropy 现在不支持axis ！= (rank - 1) 的规模，所以先不要打开单测。

**Review by fuyinno4 — APPROVED** [review]

**Review by luotao1 — APPROVED** [review]

**Comment by taixiurong** [comment]
commit fail because use root to commit
